includes/class-activitypub.php

```diff
@@ -241,6 +214,11 @@ public static function add_rewrite_rules() {
 				'index.php?rest_route=/' . ACTIVITYPUB_REST_NAMESPACE . '/nodeinfo2',
 				'top'
 			);
+			\add_rewrite_rule(
+				'^@([\w]+)',
+				'index.php?rest_route=/' . ACTIVITYPUB_REST_NAMESPACE . '/users/$matches[1]',
+				'top'
+			);
```

Comment:
This is going to allow us to do something like this?

```
https://example.com/@user@example.com
```

Reply:
The second one. To have URLs for pseudo users (like the blog user) that has no author page. It's kind of the same as on Mastodon, where the @username URLs redirect to the classic URLs.